hello/hello_activity_choice.py

```diff
@@ -59,39 +59,22 @@ async def run(self, list: ShoppingList) -> str:
         ordered: List[str] = []
         for item in list.items:
```

Comment:
A tiny thing, not related to this PR, but we should avoid using `list` as a variable name in Python.